crates/driver/src/tests/cases/order_prioritization.rs

```diff
@@ -13,21 +16,40 @@ use crate::{
 #[tokio::test]
 #[ignore]
 async fn sorting() {
+    let now = Utc::now().timestamp() as u32;
```

Comment:
Here I would expect test cases for the different sorting strategies (to convince ourselves the logic is correct)

Reply:
Do you mean different sets of sorting strategies or a separate case for each strategy?

Reply:
Yes, I assume here you are testing the default sorting strategy (which is quite implicit). i'd expect a good test to explicitly set a strategy (e.g. CreationTimestamp) and then assert the ordering. Then another test case that sets the strategy to something different and asserts its ordering. And finally maybe a test that shows how those strategies can be composed.

Reply:
Added more tests.